Jenkinsfile

```diff
@@ -0,0 +1,114 @@
+#!/bin/groovy
+@Library('SonarSource@github') _
+
+pipeline {
+  agent { 
+    label 'linux' 
+  }
+  parameters {
+    string(name: 'GIT_SHA1', defaultValue: 'master', description: 'Git SHA1 (provided by travisci hook job)')
+    string(name: 'CI_BUILD_NAME', defaultValue: 'sonar-license', description: 'Build Name (provided by travisci hook job)')	
```

Comment:
[Minor] I would change the default value to something else (more generic as you did for other values).